ShareActionExtension/Scene/ShareViewController.swift

```diff
@@ -69,16 +69,9 @@ extension ShareViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        setupTheme(theme: ThemeService.shared.currentTheme.value)
-        ThemeService.shared.apply(theme: ThemeService.shared.currentTheme.value)
-        ThemeService.shared.currentTheme
-            .receive(on: DispatchQueue.main)
-            .sink { [weak self] theme in
-                guard let self = self else { return }
-                self.setupTheme(theme: theme)
-            }
-            .store(in: &disposeBag)
-        
+        setupTheme()
+        ThemeService.shared.apply()
```

Comment:
Do you know why we have to call `ThemeService.shared.apply()` here? This method is initially invoked in the `AppDelegate`.

Reply:
Ah right, I didn't notice that it's part of the Share Extension. Makes sense then.